axon-server/administration/reset-event-processor-token.md

```diff
@@ -0,0 +1,146 @@
+    .ifPresent(streamingEventProcessor -> {
+        if (streamingEventProcessor.supportsReset()) {
+            streamingEventProcessor.shutDown();
+            streamingEventProcessor.resetTokens();
+            streamingEventProcessor.start();
+        }
+    });
+```
+
+Note that this only concerns stopping the local instances. 
+If there are instances running on other nodes, you either need to use AxonServer or build a solution for this yourself.
+For more details, refer to the section in the Axon Framework [reference guide](../../axon-framework/events/event-processors/streaming.md#triggering-a-reset).
+
+## Using the REST API
+Axon Server exposes a REST API to pause and start EventProcessors.
+When used in conjunction with the `resetTokens` method from earlier, this can be used to make sure that all instances of an EventProcessor are paused before a token is reset.
+The required parameters to do this are the following:
+- *component*: name of the component the EventProcessor is part of
+- *processor*: name of the EventProcessor itself
+- *context*: name of the context for which to reset the tokens
+- *tokenStoreId*: the identifier used to distinguish the desired token from other tokens stored in the same store. 
+
+Now 
+`/v1/components/{component}/processors/{processor}/pause?context={context}&tokenStoreIdentifier={tokenStoreId}`
+can be called, causing AxonServer to request all matched EventProcessors to stop.
+
+Then a reset as shown in the example using the framework can be performed.
+This is not Axon Server specific and hence is the same for all shown methods.
+
+As a final step, the EventProcessors can be started again with a patch request to the following url:
+`/v1/components/{component}/processors/{processor}/start?context={context}&tokenStoreIdentifier={tokenStoreId}` .
+
+
+## Using the Axon Server Connector
+To reset EventProcessors with the Axon Server Connector, the dependency has to be available on your classpath, 
+e.g. by using the following maven dependency.
+```xml
+<dependency>
+    <groupId>io.axoniq</groupId>
+    <artifactId>axonserver-connector-java</artifactId>
+    <version>4.6.0-SNAPSHOT</version>
+</dependency>
+```
+
+All operations related to administration go through an `AdminChannel`, 
+which can be obtained from an `AxonServerConnectionFactory` as follows.
+
+```java
+private AdminChannel adminChannel() {
+    AxonServerConnectionFactory connectionFactory = AxonServerConnectionFactory.forClient(componentName).build();
+    return connectionFactory.connect(contextName).adminChannel();
+}
+```
+
+In this example, `componentName` and `contextName` are values supplied by external configuration.
+In simple cases, these might be the same as in your axon-framework configuration.
+
+Using the provided admin channel, you can pause, reset and restart the event processors.
+In contrast to the approach based on the Axon Framework, you also need to provide a `tokenStoreIdentifier`, 
```

Comment:
Might be nice to tell users how they can retrieve the `tokenStoreIdentifier`. Might be good even to add this notice on the "Streaming Event Processors" page's Token Store section, as it's currently not drafted up there at all. Having a piece on it there gives you the opportunity to link to it from this page.